charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [8.0.4]
+* Update SonarQube to 
+* 9.9.4
+
 ## [7.0.3]
-* Update SonarQube to 9.9.3
```

Comment:
This changelog entry should not be modified.

Reply:
fixed ✅